test/Graphics/Transformable.test.cpp

```diff
@@ -34,11 +34,11 @@ TEST_CASE("[Graphics] sf::Transformable")
         CHECK(transformable.getPosition() == sf::Vector2f(3, 4));
 
         transformable.setRotation(sf::degrees(3.14f));
-        CHECK(transformable.getRotation() == sf::degrees(3.14f));
+        CHECK(transformable.getRotation() == Approx(sf::degrees(3.14f)));
         transformable.setRotation(sf::degrees(540));
-        CHECK(transformable.getRotation() == sf::degrees(180));
+        CHECK(transformable.getRotation() == Approx(sf::degrees(180)));
         transformable.setRotation(sf::degrees(-72));
-        CHECK(transformable.getRotation() == sf::degrees(288));
+        CHECK(transformable.getRotation() == Approx(sf::degrees(288)));
```

Comment:
Let's change the `setRotation` arguments to use `sf::radians` so we can avoid adding approximate comparisons.

Reply:
I gave it a try and some tests still failed due to small floating point precision errors. I don't think it's worthwhile spending time on avoiding the use of `Approx` in the test, I agree with @L0laapk3's technical arguments that `Approx` is what we want to test for, not an exact value.